Tools/jit/README.md

```diff
@@ -21,6 +21,12 @@ chmod +x llvm.sh
 sudo ./llvm.sh 18
 ```
 
+Install LLVM 18 on Fedora Linux 40 or newer:
+
+```sh
+sudo dnf install 'clang(major) = 18' 'llvm(major) = 18'
```

Comment:
Thanks for this! Just double-checking, is the separate Clang install necessary (it's not bundled with the LLVM install)?

Reply:
Correct. Installing just LLVM does not install clang. Neither the other way around.